### ISSUE TITLE
Not starting after updating from v2.0.4.23 to v2.0.4.25

### ISSUE THREAD

**Issue by sfahrenholz** [duplicati/duplicati#3882]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info
- **Duplicati version**: v2.0.4.25
- **Operating system**: Windows 10
- **Backend**: 

## Description
After updating to the newest canary version, duplicati does not starting

## Steps to reproduce
1. Version v2.0.4.23
2. Updating v2.0.4.25
3. Activated Update
4. ??

## Debug log
```
System.Reflection.TargetInvocationException: Ein Aufrufziel hat einen Ausnahmefehler verursacht. ---> System.AggregateException: Mindestens ein Fehler ist aufgetreten. ---> System.IO.FileLoadException: Die Datei oder Assembly "Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" oder eine Abhängigkeit davon wurde nicht gefunden. Die gefundene Manifestdefinition der Assembly stimmt nicht mit dem Assemblyverweis überein. (Ausnahme von HRESULT: 0x80131040) ---> System.IO.FileLoadException: Die Datei oder Assembly "Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" oder eine Abhängigkeit davon wurde nicht gefunden. Die gefundene Manifestdefinition der Assembly stimmt nicht mit dem Assemblyverweis überein. (Ausnahme von HRESULT: 0x80131040)
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei Duplicati.Library.UsageReporter.EventProcessor.<>c__DisplayClass8_0.<<Run>b__0>d.MoveNext()
   bei System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine)
   bei Duplicati.Library.UsageReporter.EventProcessor.<>c__DisplayClass8_0.<Run>b__0(<>f__AnonymousType0`2 self)
   bei CoCoL.AutomationExtensions.<RunTask>d__10`1.MoveNext()
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   bei System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   bei System.Threading.Tasks.Task.Wait(TimeSpan timeout)
   bei Duplicati.Library.UsageReporter.Reporter.ShutDown()
   bei Duplicati.Server.Program.RealMain(String[] _args)
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   bei System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   bei System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   bei Duplicati.Library.AutoUpdater.UpdaterManager.RunMethod(MethodInfo method, String[] args)
```

**Comment by warwickmm** [comment]
This is actively being looked into.  More discussion at the links below:

https://forum.duplicati.com/t/release-2-0-4-24-canary-2019-09-02/7914
https://forum.duplicati.com/t/release-2-0-4-25-canary-2019-09-02/7918

**Comment by warwickmm** [comment]
There was an issue with the release build that should be fixed in version 2.0.4.26.

https://forum.duplicati.com/t/release-2-0-4-26-canary-2019-09-02/7930
